### DISCUSSION TITLE
nuxt generate编译后，输出文件内部代码 包含一些es6内容。如何变成es5

### DISCUSSION THREAD

**Discussion by zxzxac110** [nuxt/nuxt#28677] [General]
`"nuxt": "^3.12.4" `使用 `nuxt generate` 后 输出文件内部代码 包含一些es6内容 
比如 `<script type="module"` 导致在chrome56中跑不起来

我使用 
`"nuxt-vite-legacy": "^1.2.0",`
在`nuxt.config.ts`中配置
```
  vite: {
    plugins: [
      legacy({
        targets: ["chrome 49"],
        renderLegacyChunks: false,
        modernPolyfills: ['es.global-this', 'es/object', 'es/array']
      }),
    ],
  },
```
generate 后依然还是出现es6内容 导致在chrome56中依旧跑不起来
如何使其转变成es5代码

**Comment by xjccc** [comment]
最低支持ES6呢

**Reply by zxzxac110** [reply]
那对于低版本的要放弃 或者 换其他技术吗

**Reply by xjccc** [reply]
我觉得。。绝大部分可以放弃一些低版本浏览器了吧。主流的基本很难支持低版本浏览器的吧。nuxt2应该可以、就是后续不会有更新了

**Reply by zxzxac110** [reply]
打工仔没资格表示只能咬牙转nuxt2了

**Comment by itshizhan** [comment]
需要转es5的，为什么用jQuery 1.x 呢
Vue、React 都是为现代浏览器而生的

**Reply by zxzxac110** [reply]
因为 开始 不知道
